packages/node/src/api-router.ts

```diff
@@ -31,6 +31,41 @@ import {
 } from "./methods";
 import { NODE_EVENTS } from "./types";
 
+// Maps controllers to JSONAPI operations.
+export const controllersToOperations = {
```

Comment:
Why are some ops in a fixed set like `get` or `add` but others are strings like `getFreeBalanceState`?

Reply:
Is there an intro to `@ebryn/jsonapi-ts` with motivation?